BookingSpecTests/SampleTest.feature

```diff
@@ -3,9 +3,122 @@
 	As a math idiot
 	I want to be told the sum of two numbers
 
-Scenario: Open Main Page and Click Sign In Button
+Scenario Template: Attempt to Sign In using Invalid combinations of email and password
 	Given I open browser
 	And I navigate to url "https://booking.com"
-	Then I wait for 5 seconds
 	When I click Sign In button
-	Then I wait for 5 seconds
\ No newline at end of file
+	And I set following parameters on Sign In Pop Up dialog
+		| Field    | Value      |
+		| Email    | <Email>    |
+		| Password | <Password> |
+	And I click Sign In button on PopUp
+	Then I wait while page popup is working
+	And I see message error with "<Message>"
+
+Scenarios: 
+		| Email                      | Password  | Message                                                               |
+		| testmirantistest@gmail.com | wrongpass | You entered an email address/password combination that doesn't match. |
+		| invalidEmail@g.com         | 1234qweR  | You entered an email address/password combination that doesn't match. |
+		| invalidEmail@g.com         | wrongpass | You entered an email address/password combination that doesn't match. |
+		|                            | 1234qweR  | Please enter a valid email address.                                   |
+		| testmirantistest@gmail.com |           | Please add a password                                                 |
+		|                            |           | Please enter a valid email address.                                   |
+
+	Scenario: Input valid credintials to Sign In
+	Given I open browser
+	And I navigate to url "https://booking.com"
+	When I click Sign In button
+	And I write email on Sign in PopUp "testmirantistest@gmail.com"
+	And I write password "1234qweR"
```

Comment:
I think it will be better to use already existing step for it:
'And I set following parameters on Sign In Pop Up dialog'